doc/zh-cn/plugins/prometheus.md

```diff
@@ -120,12 +120,49 @@ plugin_attr:
 
 ### 可有的指标
 
-* `Status codes`: upstream 服务返回的 HTTP 状态码，每个服务返回状态码的次数或者所有服务的状态码次数总和都可以统计到。
-* `Bandwidth`: 流经apisix的总带宽(可分出口带宽和入口带宽). 每个服务指标或者是所有服务指标的总和都可以统计到。
+* `Status`: upstream 服务返回的 HTTP 状态码，每个服务返回状态码的次数或者所有服务的状态码次数总和都可以统计到。具有的维度：
+
+    | 名称          |    描述             |
+    | -------------| --------------------|
+    | code         | upstream 服务返回的 HTTP 状态码。 |
+    | route        | 请求匹配的 route 的 `route_id`，未匹配，则默认为`""`。 |
```

Comment:
the default value is empyy string or what else？

Reply:
Yes, in the case of no matching route, the default value is an empty string.

like this:

```
apisix_http_status{code="404",route="",matched_uri="",matched_host="",service="127.0.0.1",consumer="",node=""} 1
```